app/views/layouts/admin.html.erb

```diff
@@ -3,23 +3,26 @@
 <head>
   <meta charset="utf-8">
   <meta name="ROBOTS" content="NOODP" />
-  <%= stylesheet_link_tag    'application', media: 'all', debug: true %>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
Not really blocking this, but I wonder if we have any policy about using third-party CDNs or not